Engine/Main/Application.cpp

```diff
@@ -62,6 +62,16 @@ Application::~Application()
     }
 }
 
+void Application::Import()
+{
+
+	resources->Init();
+	while (!resources->thread_comunication.finished_loading)
+	{
+		std::cout << "vblablabla porcentaje blablabla";
+	}
+}
+
```

Comment:
Better remove this that is very cutre xD

Reply:
Ok! I totally forgot about it :D